src/components/simple-color-picker/simple-color-picker.scss

```diff
@@ -1,5 +1,6 @@
 .carbon-simple-color-picker {
   width: 300px;
+  padding-bottom: 60px;
```

Comment:
what is this needed for?

Reply:
there was a bug in previous storybook version, unnecessary padding is deleted now